Automattic/jetpack
# Adjust copy on Connection banner

## Proposed changes:

* Change the logic of `useConnectionWatcher` to always display the "user connection" message, since the connection interstitial always mentions user connection. This fix is just to make it consistent with what we are currently displaying to the user. The plan is to fix the flow in the future.

### Other information:


## Jetpack product discussion


See linked issue

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Spin up a JN site with Jetpack Beta pointing to this branch
* Without connecting, go to My Jetpack and dismiss the Welcome banner
* Refresh the page and confirm you see a notice like the one below:

<img width="1106" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/edcd691e-c40f-407f-b3cb-b44db05fad22">
